src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java

```diff
@@ -80,14 +80,14 @@ public ConfigScreenFactory<?> getModConfigScreenFactory() {
 					.build());
 
 			// Set an option for verifiedIconOffsetX
-			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetX"), NCRConfig.getClient().verifiedIconOffsetX, -5, 5)
+			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetX"), NCRConfig.getClient().verifiedIconOffsetX, -10, 10)
```

Comment:
What, why so little? Make it 1k at least.

Reply:
https://user-images.githubusercontent.com/17520451/187282060-47547f7f-145b-4a7a-b237-e07b06080584.mp4

it makes it too hard to control if you have it too much, i could remove the slider and let it be a input

Reply:
Then make it an input.

Reply:
Done